src/api/middleware/validators/auth.js

```diff
@@ -0,0 +1,10 @@
+const { celebrate, Joi } = require("celebrate");
```

Comment:
We should switch to [`express-validator`](https://www.npmjs.com/package/express-validator) because of this: https://www.npmtrends.com/celebrate-vs-express-validator